spec/lexers/console_spec.rb

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*- #
+
+describe Rouge::Lexers::ConsoleLexer do
+  let(:subject) { Rouge::Lexers::ConsoleLexer.new }
+
+  describe 'lexing' do
+    include Support::Lexing
+
+    it 'recognizes multilines which end with $' do
+      assert_tokens_equal "--foo \\\n--bar $",
+        ["Generic.Prompt", "--foo \\\n--bar $"]
+    end
+  end
+end
```

Comment:
The "doesn't drop text" invariant from `spec/lexers_spec.rb` should be in effect if you add this sample to the visual spec.